mcsema/Arch/X86/Semantics/String.cpp

```diff
@@ -859,4 +864,7 @@ void String_populateDispatchMap(DispatchMap &m) {
   m[llvm::X86::REPNE_CMPSW_64] = translate_CMPS16;
   m[llvm::X86::REPNE_CMPSD_64] = translate_CMPS32;
   m[llvm::X86::REPNE_CMPSQ_64] = translate_CMPS64;
+
+  m[llvm::X86::REP_PREFIX] = translate_REP_prefix;
```

Comment:
So I moved this here because we had a special case in the instruction lifting to detect and suppress errors when `REP` isn't handled. In an attempt to make that more arch-neutral, and get the same behavior, I added these nop translators.

Things are handled in a few weird ways. There are a bunch of REP_-prefixed LLVM opcodes in 3.8 that we use. We also have internal handling of REP-related prefixes in the NativeInst class. All of these are used.

In general. we should never see two separate instructions, and before when we did we suppressed the issue, just in a different way.